cli/src/cli_util.rs

```diff
@@ -260,10 +260,23 @@ impl TracingSubscription {
     pub fn enable_debug_logging(&self) -> Result<(), CommandError> {
         self.reload_log_filter
             .modify(|filter| {
+                // The default is INFO.
+                // jj-lib and jj-cli are whitelisted for DEBUG logging.
+                // This ensures that other crates' logging show up by default.
                 *filter = tracing_subscriber::EnvFilter::builder()
-                    .with_default_directive(tracing::metadata::LevelFilter::DEBUG.into())
+                    .with_default_directive(tracing::metadata::LevelFilter::INFO.into())
                     .with_env_var(Self::ENV_VAR_NAME)
-                    .from_env_lossy();
+                    .from_env_lossy()
+                    .add_directive(
+                        "jj_lib=debug"
+                            .parse()
+                            .expect("directive should be well formed"),
+                    )
+                    .add_directive(
+                        "jj_cli=debug"
+                            .parse()
+                            .expect("directive should be well formed"),
```

Comment:
BTW, it's probably fine to just use `.unwrap()` here. I don't think this message helps much with either readability or debuggability.